client/components/lists/listHeader.js

```diff
@@ -47,6 +49,70 @@ Template.listActionPopup.events({
   },
 });
 
+Template.listImportCardPopup.events({
+  submit(evt) {
+    // 1. get the json data out of the form and parse it
+    evt.preventDefault();
+    const jsonData = $(evt.currentTarget).find('textarea').val();
+    const data = JSON.parse(jsonData);
```

Comment:
I'm having trouble accessing the `error` field (this.error is undefined in my method) - where is it defined?